README.md

```diff
@@ -121,3 +121,15 @@ from sdk.device.configuration import EmuFreeConfig
 configuration = EmuFreeConfig(with_noise=True)
 batch = sdk.create_batch(serialized_sequence, [job1,job2], device_type=DeviceType.EMU_FREE, configuration=configuration)
 ```
+
+### List of supported device specifications
+
+The SDK provides a method to retrieve the device specs supported by Pasqal.
+
+```python
+sdk.get_device_specs_list()
+```
+
+The method returns a list of Pydantic `DeviceSpecs` objects with two parameters: `device_type` and `specs`. The `specs` are
+in the `dict` format. They can be dumped into a json string and deserialized into a `Device` instance from the `Pulser`
+library with the `deserialize_device` method from `Pulser`.
```

Comment:
there is the PulserPasqal module in Pulser. 

I'm not sure where to but the boundary between the packages but perhaps such an example (or even convenience function) belongs in Pulser and not in the cloud-sdk?


Fully agree on pointing to the relevant docs though

Reply:
Yeah for me the boundary is also very unclear as well

Reply:
As we discussed in person @MatthieuMoreau0 , the sequence validate against the device specs is intrinsic and happens every time the sequence is constructed. If you are coming from a serialized sequence, the deserialization will rebuild that sequence with the given device and during this process all the necessary validation will occur (including the register validation, by the way).

Reply:
I thought it could be useful for sdk users to understand that they could use these specs as such to build their device in the pulser lib. I understand the sdk is a different tool but it doesn't really serve a purpose without the pulser library anyway. But I can remove this comment if we want to keep things separated.

Reply:
I think you should leave it as is, and we should add something on the PulserPasqal side instead?